cmd/utils/flags.go

```diff
@@ -456,6 +456,11 @@ var (
 		Usage: "First block to start syncing from on the L1",
 		Value: 0,
 	}
+	L1ContractAddressCheckFlag = cli.BoolFlag{
```

Comment:
hmm, where did this flag come from?

Reply:
This one lets us turn off the address checks - useful if you want to run devnet locally